wpimath/src/main/native/cpp/util/PIDConstants.cpp

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#include "frc/util/PIDConstants.h"
+
+#include "wpimath/MathShared.h"
+
+frc::PIDConstants::PIDConstants(double Kp, double Ki, double Kd) {
+  this->Kp = Kp;
+  this->Ki = Ki;
+  this->Kd = Kd;
```

Comment:
For consistency, do what PIDController does by putting `using namespace frc;` near the top of the file instead of using `frc::`.
```suggestion
PIDConstants::PIDConstants(double Kp, double Ki, double Kd) : Kp{Kp}, Ki{Ki}, Kd{Kd} {
```